### ISSUE TITLE
Why can't gnuroot debian allow network support?

### ISSUE THREAD

**Issue by Root-voidX** [corbinlc/GNURootDebian#231]
I can't even use firefox to surf the web in xterm. Is there anything I need to install/configure for internet connection?

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by Root-voidX** [comment]
Yes, if possible can you give network access to gnuroot debian? Termux proot gives network access so it be nice to get it for gnuroot debian so we can actually use and download packages via browsers like firefox

**Comment by ExtremeNerd** [comment]
What? Doesn't it use network when u installed Firefox from Debian repo? Maybe a reinstallation of package or installing it from some other source should work..

**Comment by AdamMatthewLTM** [comment]
Hi all,

My apologies for the delay, and thank you for your replies. @Root-voidX , I too have some difficulties in accessing some network capabilities, and am therefore recommending this issue remain open as medium priority until further notice.

Thank you for your patience,
-Adam
